### DISCUSSION TITLE
Add highlighting support for webpages

### DISCUSSION THREAD

**Discussion by PalmerAL** [minbrowser/min#1496] [Ideas]
Selecting text on a page and right-clicking it would let you highlight that text to remember it later. This would be useful for any kind of research task in the browser, where you want to read a lot of information and save important parts.

Other details:
- Highlights would be saved for as long as the page remains in bookmarks / history
- Highlights would be searchable, and would appear with the page in the searchbar

Uniquely identifying text fragments within pages is generally a hard problem, but I think most of it could be handled by this library: https://github.com/GoogleChromeLabs/text-fragments-polyfill

**Comment by shalva97** [comment]
checked out the branch. it looks cool but I dont know.... thats something not for me...

at least please put Add highlight at the bottom of the menu, below Copy would be fine too. I hate it when the order changes, more than half year of my reflexes would be hard to unlearn

**Reply by PalmerAL** [reply]
Yeah, that makes sense, I'll change it.
